### PR TITLE
Initial project page

### PR DESCRIPTION
**Pull request by qinqon**
**Is this a BUG FIX or a FEATURE ?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What this PR does / why we need it**:
It does not use docker and it's more flexible.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by qinqon** [comment]
/hold

We should have a review CI at GCS bucket or the like.

File: .github/workflows/jekyll.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      - jekyll
+      # - source
+      # It is highly recommended that you only run this action on push to a
+      # specific branch, eg. master or source (if on *.github.io repo)
+
+jobs:
+  jekyll:
+    runs-on: ubuntu-16.04 # can change this to ubuntu-latest if you prefer
+    steps:
+      - name: 📂 setup
```
Comment (phoracek) [inline .github/workflows/jekyll.yaml:16]: That's fancy!

File: .github/workflows/jekyll.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+name: Build and deploy jekyll site
+
+on:
+  push:
+    branches:
+      - master
+      - jekyll
```
Comment (phoracek) [inline .github/workflows/jekyll.yaml:7]: We don't need this, do we?

File: .github/workflows/jekyll.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      # - source
+      # It is highly recommended that you only run this action on push to a
```
Comment (phoracek) [inline .github/workflows/jekyll.yaml:9]: Please drop all boilerplate comments

File: docs/_config.yaml [hunk 4]
```diff
@@ -1,3 +1,30 @@
-# _config.yml
-
-title: "Jekyll Actions Demo"
+theme: minimal-mistakes-jekyll
+minimal_mistakes_skin: "air"
+locale: "en-US"
+title: "The Kubernetes-nmstate project"
```
Comment (phoracek) [inline docs/_config.yaml:4]: "Kubernetes NMState" or "kubernetes-nmstate" would be enough IMO

File: docs/_config.yaml [hunk 5]
```diff
@@ -1,3 +1,30 @@
...
+description: A Declarative API for Kubernetes Node Network Management
+name: "The Kubernetes nmstate team"
```
Comment (phoracek) [inline docs/_config.yaml:6]: "Kubernetes NMState" or "kubernetes-nmstate"

File: docs/_data/navigation.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+main:
+  - title: User Guide
+    children:
+      - title: "Reporting"
+        url: /user-guide-101-reporting
+      - title: "Configuration"
+        url: /user-guide-102-configuration
+      - title: "Troubleshooting"
+        url: /user-guide-103-troubleshooting
+  - title: Deployment
```
Comment (phoracek) [inline docs/_data/navigation.yaml:10]: I'd put deployment first

File: docs/index.md [hunk 7]
```diff
@@ -1,9 +1,73 @@
 ---
+title: "The Kubernetes nmstate project"
```
Comment (phoracek) [inline docs/index.md:2]: This is already in the page title. Here it can be "Introduction"

**Review by phoracek — CHANGES_REQUESTED** [review]
Use symlinks or copy for the logo if you need. The repository should keep our logo directory visible, so we know where to find it. It is not meant only as a documentation asset.

Other than that, some nits. I think this is a good start.

**Comment by qinqon** [comment]
@phoracek moved the PR to https://github.com/nmstate/kubernetes-nmstate/pull/621 to do it from normal PR user now that we have a proper preview mechanism we don't need to push to a specific branch at origin.

I will address comments there.

**Pull request closed without merging**